solution-jivanov/dynamodb.yml

```diff
@@ -0,0 +1,15 @@
+Resources:
+
+  UsersTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: ${self:custom.usersTableName}
+      AttributeDefinitions:
+      - AttributeName: id
+        AttributeType: S
+      KeySchema:
+      - AttributeName: id
+        KeyType: HASH
+      ProvisionedThroughput:
+        ReadCapacityUnits: 1
```

Comment:
If capacity provisioning is defined in Serverless spec, how would you approach dynamic autoscale needs for Dynamo tables?

Reply:
I put 1 here primarily for cost concerns, have not planned for autoscaling on this particular task. But that certainly can be done, there are (at least) 2 ways to do it: 

- We can define the table to be "on-demand" (and AWS will happily accommodate our loads). 

- OR we can add auto-scaling to our DynamoDB table (Serverless even has an autoscaling plugin which allows us to control the auto-scaling policy as IaC). 

The primary difference between the 2 is "control". The latter gives us more control over the auto-scaling.